### ISSUE TITLE
Make new examples

### ISSUE THREAD

**Issue by JelleAalbers** [XENON1T/hax#39]
The XENON100 examples we used at the analysis workshops are out of date by now; it would be nice to distill a few new examples from some of the notes that people have published.

**Comment by darrylmasson** [comment]
What kind of examples are we looking for? hax usage in general or how to do specific things?

**Comment by JelleAalbers** [comment]
Both would be very useful :-) But most questions I get are about selecting runs (tags selection, using hax.runs.datasets with pandas queries), inspecting waveforms and using the trigger data.

**Comment by darrylmasson** [comment]
I threw [this note](https://xecluster.lngs.infn.it/dokuwiki/lib/exe/fetch.php?media=xenon:xenon1t:darryl:runs_db_interface.html) together quickly for runs database interfacing, which hopefully deals with the first item on the list. I also have a note or two somewhere with slightly expanded cuts examples that more or less combines Jelle's example on hax.cuts with the preselection cuts, and one on the basic analysis procedure of setting up a notebook, loading data, making plots, and that sort of thing. I originally wrote most of them to give our legion of undergrads something to start from, but they seem like the kind of thing it may be worth brushing up a bit and adding to the hax examples.

**Comment by JelleAalbers** [comment]
Nice! This will be very useful for new analysts and people supervising new analysts (whether these come in legions or not). I've never used regex pattern matching for tags, good to see it actually works ;-) If you make a pull request I will merge it into master. Your other notebooks are also very welcome, right now we only have a few outdated general examples and some examples showing off specific functionality. 

If you're looking for a thing or two to add to this example:
  * You can chain tags selection to require the presence of more than one tag, e.g. : 
```
tsel = hax.runs.tags_selection
tsel(tsel(include='sciencerun*'), include='unblinded')
```
  * If you want to view all tags combinations you have in dsets, you can do `np.unique(dsets.tags)`
  * A few words on the pax version policy option might be useful, or just a link to https://github.com/XENON1T/hax/pull/30 to make new people aware such a thing exists (although they won't use it in the first month probably). If you use things like location != "" this can control whether or not a dataset appears in your selection.

More seasoned analysts might also like to know:
  * The rather arcanely named field reader__ini__name in hax.runs.datasets is actually the run mode name, which can be useful for selection as well (e.g. if you want to select all 1ms noise runs). 
  * Probably overkill for beginners, but hax.runs.get_run_info can also be very useful (e.g. `hax.runs.get_run_info([6000, 6001], 'processor.DEFAULT.electron_lifetime_liquid')`)

**Comment by JelleAalbers** [comment]
Fixed in #224
